doc/installation.rst

```diff
@@ -383,7 +383,7 @@ Testing
 
 If you want to run the tests::
 
-    sudo pip install nose
+    sudo pip install pytest nose
```

Comment:
Do you really still need to install both pytest and nose?

Reply:
Unfortunately yes...  Until we have Numpy 1.15 as a minimal version.  `numpy.testing` depends on `nose` until that version